sui/src/benchmark.rs

```diff
@@ -100,32 +105,25 @@ fn run_throughout_microbench(
     // In order to simplify things, we send chunks on each connection and try to ensure all connections have equal load
     assert!(
         (num_transactions % connections) == 0,
-        "num_transactions must {} be multiple of number of TCP connections {}",
+        "num_transactions must be a multiple of number of TCP connections {}, got {}",
+        connections,
         num_transactions,
-        connections
     );
-    let mut tx_cr = TransactionCreator::new(committee_size, db_cpus);
+    let mut tx_cr = TransactionCreator::new();
 
     let chunk_size = batch_size * connections;
     let txes = tx_cr.generate_transactions(
         connections,
         use_move,
         batch_size * connections,
         num_transactions / chunk_size,
+        &mut validator_preparer,
     );
 
-    // Make multi-threaded runtime for the authority
-    thread::spawn(move || {
-        get_multithread_runtime().block_on(async move {
-            let server = spawn_authority_server(network_server, tx_cr.authority_state).await;
-            if let Err(e) = server.join().await {
-                error!("Server ended with an error: {e}");
-            }
-        });
-    });
+    validator_preparer.deploy_validator(network_server);
 
     // Wait for server start
-    sleep(Duration::from_secs(3));
+    sleep(Duration::from_secs(20));
```

Comment:
can we keep this low for the case of a same process bench? It makes a difference when profiling to not wait for 1/2 minute.

Reply:
forgot to revert this change, thanks for calling out